lib/rest.js

```diff
@@ -360,6 +360,61 @@ class BinanceRest {
         return this._makeRequest(query, callback, 'api/v3/allOrders', 'SIGNED');
     }
 
+    newOCO(query = {}, callback) {
+        this._setTimestamp(query);
+        return this._makeRequest(
+            query,
+            callback,
+            'api/v3/order/oco',
+            'SIGNED',
+            'POST'
+        );
+    }
+
+    cancelOCO(query = {}, callback) {
+        this._setTimestamp(query);
+        return this._makeRequest(
+            query,
+            callback,
+            'api/v3/orderList',
+            'SIGNED',
+            'DELETE'
+        );
+    }
+
+    queryOCO(query = {}, callback) {
+        this._setTimestamp(query);
+        return this._makeRequest(query, callback, 'api/v3/orderList', 'SIGNED');
+    }
+
+    queryAllOCO(query = {}, callback) {
+        if (_.isFunction(query)) {
+            callback = query;
+            query = {};
+        }
```

Comment:
Looks good so far! Small observation, is there a reason for these reassignments? I would prefer to remove them to keep things simpler. If the first param isn't desired, you could just pass undefined or null.

Reply:
Sure we can pass in undefined or null, but _.isFunction() is being used in other part of the API's also where all the **params** are optional. So I thought to include it here also. 
Because in any way, even if a user passes in a null value or does passes the first parameter of the method, in both cases the code would run just fine.

Or If you want I can remove these reassignments also.

Reply:
Sorry for the delay - I'm happy to merge this as is. I'm working on a revamp of this library to make way for all the missing endpoints and the other parts of binance APIs, so no need to make major changes in how the current version of the library worked.